tests/index/test_find2_index_approx.h

```diff
@@ -36,6 +36,7 @@
 #define TESTS_FIND2_INDEX_APPROX_H_
 
 #include <mutex>
+#include <time.h>
```

Comment:
Never use C-headers directly if there are are C++ headers with namespace std (`<ctime>`).

Please have a look how random is done in c++11:
http://en.cppreference.com/w/cpp/numeric/random/rand